json-path/src/main/java/com/jayway/jsonpath/internal/function/json/Unique.java

```diff
@@ -0,0 +1,56 @@
+package com.jayway.jsonpath.internal.function.json;
+
+import com.jayway.jsonpath.internal.EvaluationContext;
+import com.jayway.jsonpath.internal.PathRef;
+import com.jayway.jsonpath.internal.function.Parameter;
+import com.jayway.jsonpath.internal.function.PathFunction;
+import net.minidev.json.JSONArray;
+
+import java.util.*;
```

Comment:
`import java.util.*; `
Could we avoid  import all, and only import the necessary  classes here   ?